<launch>
  <arg name="settings_common" default="$(find hri_planner)/resources/planner_setting/exp_settings_common.yaml" />

  <node name="planner_node" pkg="hri_planner" type="planner_node" output="screen">
    <rosparam command="load" file="$(arg settings_common)" />
	<remap from="/planner/cmd_vel" to="/cmd_vel_mux/input/navi" />
  </node>
</launch>
